platform/src/common/components/Map/components/Sidebar.js

```diff
@@ -658,22 +741,25 @@ const Sidebar = ({ siteDetails, selectedSites, isAdmin, showSideBar, setShowSide
                 />
               </div>
 
-              <LocationDetailItem
-                title='Air Quality Alerts'
-                isCollapsed
-                children={
-                  <p className='text-xl font-bold leading-7 text-secondary-neutral-dark-950'>
-                    <span className='text-blue-500 capitalize'>
-                      {selectedSite.place_name ||
-                        selectedSite.name.split(',')[0] ||
-                        selectedSite.search_name}
-                      's
-                    </span>{' '}
-                    Air Quality is expected to be Good today. Enjoy the day with confidence in the
-                    clean air around you.
-                  </p>
-                }
-              />
+              {selectedSite?.airQuality && (
+                <LocationDetailItem
+                  title='Air Quality Alerts'
+                  isCollapsed
+                  children={
+                    <p className='text-xl font-bold leading-7 text-secondary-neutral-dark-950'>
+                      <span className='text-blue-500 capitalize'>
+                        {selectedSite?.place_name ||
+                          selectedSite?.name?.split(',')[0] ||
+                          selectedSite?.search_name ||
+                          selectedSite?.location}
+                        's
```

Comment:
this "'s" looks suspicious, is it supposed to be there?

Reply:
It's part of the text. The location name followed by 's